src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    static final int NUMBER_OF_FIGURES = 5;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[NUMBER_OF_FIGURES];
+
+        for (int i = 0; i < NUMBER_OF_FIGURES; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            System.out.println(figures[i].draw());
+        }

```

Comment:
```suggestion
        for (int i = 0; i < NUMBER_OF_FIGURES; i++) {
            if (i < NUMBER_OF_FIGURES / 2) {
                figures[i] = figureSupplier.getRandomFigure();
            } else {
                figures[i] = figureSUpplier.getDefaultFigure();
            }
            System.out.println(figures[i].draw());
        }
```
you task was also to create a method that returns default figure

Reply:
not fixed